ios/build/ios/XCBuildData/PIFCache/target/TARGET@v11_hash=39047dbfca6f81dc03f9b20d0e79586f-json

```diff
@@ -0,0 +1 @@
ad908a4c6da027d1a20d80"},{"fileReference":"bfdfe7dc352907fc980b868725387e98df667679e56e70eaf2d7c800775325db","guid":"bfdfe7dc352907fc980b868725387e98f0d4b0d451c439d3385a22af61cd8150"},{"fileReference":"bfdfe7dc352907fc980b868725387e989b02ba4616a0f2c6e5b3280da650a4ea","guid":"bfdfe7dc352907fc980b868725387e98185b78f0cf6caaef67807df272c9f21d"},{"fileReference":"bfdfe7dc352907fc980b868725387e98b070267723be01e22ca7ef3c9eccd6f4","guid":"bfdfe7dc352907fc980b868725387e98f9a7c020b13c965118886166cecd6cb0"},{"fileReference":"bfdfe7dc352907fc980b868725387e987d879c3655de3da5e8d7004288a022c3","guid":"bfdfe7dc352907fc980b868725387e9895970dabe9d37e78e41d90335947346b"},{"fileReference":"bfdfe7dc352907fc980b868725387e984385732af444fc889cd19dd7a5e22db0","guid":"bfdfe7dc352907fc980b868725387e98be41e26666b304792c03116d24840ac7"},{"fileReference":"bfdfe7dc352907fc980b868725387e98004eab4da8eb8be39a5f8a79fafd5f5b","guid":"bfdfe7dc352907fc980b868725387e985b778a9edd97857c42e7896c00d25d97"},{"fileReference":"bfdfe7dc352907fc980b868725387e98f40764c29103103eb309188918086da6","guid":"bfdfe7dc352907fc980b868725387e9852d34736d737c36e1c95db048db5f429"},{"fileReference":"bfdfe7dc352907fc980b868725387e982955634dca1674250e26b4d59297f56c","guid":"bfdfe7dc352907fc980b868725387e98a94581a835c6d4ea99a744a561e2aa62"}],"guid":"bfdfe7dc352907fc980b868725387e98667120ff0eab51095e0dbd5d9e9e0137","type":"com.apple.buildphase.sources"},{"buildFiles":[{"fileReference":"bfdfe7dc352907fc980b868725387e981851f400a3440227ea96cb082236b4c9","guid":"bfdfe7dc352907fc980b868725387e98ecb730fdd540bd3799d1e8a144dc4ede"}],"guid":"bfdfe7dc352907fc980b868725387e983d2adf390cb11fb15ab630f8cc7b26b2","type":"com.apple.buildphase.frameworks"},{"buildFiles":[{"guid":"bfdfe7dc352907fc980b868725387e986a44e14377ed7be998f269b31929c7ec","targetReference":"bfdfe7dc352907fc980b868725387e9890f568fc9b811cd56b08e401eec0f35e"}],"guid":"bfdfe7dc352907fc980b868725387e981fc8e779d69bcb3e6b4cbb0e393c16a6","type":"com.apple.buildphase.resources"}],"buildRules":[],"dependencies":[{"guid":"bfdfe7dc352907fc980b868725387e989da425bb6d6d5d8dbb95e4afffb82217","name":"Flutter"},{"guid":"bfdfe7dc352907fc980b868725387e9890f568fc9b811cd56b08e401eec0f35e","name":"rive_common-rive_common_privacy"}],"guid":"bfdfe7dc352907fc980b868725387e9849c5bdbe31e3b466d37bab9271baf60a","name":"rive_common","predominantSourceCodeLanguage":"Xcode.SourceCodeLanguage.C-Plus-Plus","productReference":{"guid":"bfdfe7dc352907fc980b868725387e9884d4dff26bf6b3e66f738069c7476896","name":"rive_common.framework","type":"product"},"productTypeIdentifier":"com.apple.product-type.framework","provisioningSourceData":[{"bundleIdentifierFromInfoPlist":"${PRODUCT_BUNDLE_IDENTIFIER}","configurationName":"Debug","provisioningStyle":1},{"bundleIdentifierFromInfoPlist":"${PRODUCT_BUNDLE_IDENTIFIER}","configurationName":"Profile","provisioningStyle":1},{"bundleIdentifierFromInfoPlist":"${PRODUCT_BUNDLE_IDENTIFIER}","configurationName":"Release","provisioningStyle":1}],"type":"standard"}
```

Comment:
Are all these files required?

Reply:
It runs fine, I think it should be same.
<img width="1334" alt="Screenshot 2024-12-01 at 10 46 26 PM" src="https://github.com/user-attachments/assets/92807461-abe7-4e2a-b1d8-d992b2e9c882">